docs/content/en/docs/contribution-guide/_index.md

```diff
@@ -12,33 +12,42 @@ have an environment capable of testing changes to the Tetragon source code,
+2. [Fork the Tetragon repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)
+   to your GitHub user or organization. The repository is available under
+   [github.com/cilium/tetragon](https://github.com/cilium/tetragon).
 
-3. Turn off GitHub actions for your fork as described in the
-   [GitHub Docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#managing-github-actions-permissions-for-your-repository>).
+3. (Optional) [Turn off GitHub actions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#about-github-actions-permissions-for-your-repository)
+   for your fork. This is recommended to avoid unnecessary CI notification
+   failures on the fork.
 
-   This is recommended to avoid unnecessary CI notification failures on the fork.
-
-4. Clone your `${YOUR_GITHUB_USERNAME_OR_ORG}/tetragon` fork into your
-   `GOPATH`, and set up the base repository as `upstream` remote:
+4. [Clone your fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+   and set up the base repository as `upstream` remote:
 
    ```shell
-   mkdir -p "${GOPATH}/src/github.com/cilium"
-   cd "${GOPATH}/src/github.com/cilium"
    git clone https://github.com/${YOUR_GITHUB_USERNAME_OR_ORG}/tetragon.git
    cd tetragon
    git remote add upstream https://github.com/cilium/tetragon.git
    ```
 
-5. Prepare your [Development setup](/docs/contribution-guide/development-setup),
-   see section below.
+5. Prepare your [development setup]({{< ref "/docs/contribution-guide/development-setup" >}}).
+
+6. Check out GitHub [good first issues](https://github.com/cilium/tetragon/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+   to find something to work on. If this is your first Tetragon issue, try to
+   start with something small that you think you can do without too much
+   external help. Also avoid assigning too many issues to yourself (see [Don't
+   Lick the Cookie!](https://www.redhat.com/en/blog/dont-lick-cookie)).
+
+7. Follow the steps in [making changes]({{< ref "/docs/contribution-guide/making-changes" >}})
+   to start contributing.
+
+8. Learn how to [run the tests]({{< ref "/docs/contribution-guide/running-tests" >}}).
 
-6. Check the GitHub issues for [good tasks to get
-   started](https://github.com/cilium/tetragon/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+9. Learn how to [submit a pull request]({{< ref "/docs/contribution-guide/submitting-a-pull-request" >}})
+   to the project.
 
-7. Follow the steps in [Making changes](/docs/contribution-guide/making-changes) to start contributing. Welcome :)!
+10. Welcome to Tetragon! :)
```

Comment:
nit: should "Welcome to Tetragon!" be the first item? :)

Reply:
I changed that ☺️